log/logging.go

```diff
@@ -250,6 +264,17 @@ func (s *Logger) Debug(description string, keysAndValues ...interface{}) {
 	logMessage(s.l, s.ID, "DEBUG", description, keysAndValues...)
 }
 
+// Trace outputs an info message with an optional list of key/value pairs.
+//
+// If LogLevel is set below LevelTrace, calling this method will yield no
+// side effects.
+func (s *Logger) Trace(description string, keysAndValues ...interface{}) {
+	if s.Level < LevelTrace {
+		return
+	}
+	logMessage(s.l, s.ID, "TRACE", description, keysAndValues...)
```

Comment:
:nail_care: use a constants for these names but is doesn' t looks like we are today. I am guessing this line and line 166 need to be the same string.